compute/instances/create-start-instance/bulk_insert.go

```diff
@@ -28,37 +28,47 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
-func bulkInsertInstance(w io.Writer, projectID, zone string, template *computepb.InstanceTemplate, count int64, namePattern string, minCount *wrapperspb.Int64Value, labels map[string]string) ([]*computepb.Instance, error) {
+func createFiveInstances(w io.Writer, projectID, zone, templateName, namePattern string) ([]*computepb.Instance, error) {
 	// projectID := "your_project_id"
 	// zone := "us-central1-a"
-	// template := &computepb.InstanceTemplate{...}  // Fill in template details
-	// count := int64(5)
+	// templateName := "your_instance_template_name"
 	// namePattern := "instance-####"
-	// minCount := wrapperspb.Int64Value{Value: 3}  // Optional
-	// labels := map[string]string{"key": "value"}  // Optional
 
+	// Get instance template
 	ctx := context.Background()
-	client, err := compute.NewInstancesRESTClient(ctx)
+	client, err := compute.NewInstanceTemplatesRESTClient(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("NewInstancesRESTClient: %w", err)
+		return nil, fmt.Errorf("NewInstanceTemplatesRESTClient: %w", err)
 	}
 	defer client.Close()
 
-	if minCount == nil {
-		minCount = &wrapperspb.Int64Value{Value: count}
+	req := &computepb.GetInstanceTemplateRequest{
+		Project:          projectID,
+		InstanceTemplate: templateName,
+	}
+
+	template, err := client.Get(ctx, req)
+	if err != nil {
+		return nil, fmt.Errorf("getInstanceTemplate: %w", err)
+	}
+
+	// Initialize Instances REST Client
+	clientInstances, err := compute.NewInstancesRESTClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("NewInstancesRESTClient: %w", err)
 	}
+	defer clientInstances.Close()
 
+	// Prepare Bulk Insert Request
+	minCount := &wrapperspb.Int64Value{Value: 5}
 	bulkInsertResource := &computepb.BulkInsertInstanceResource{
 		SourceInstanceTemplate: proto.String(template.GetSelfLink()),
-		Count:                  proto.Int64(count),
+		Count:                  proto.Int64(5),
 		MinCount:               proto.Int64(minCount.GetValue()),
 		NamePattern:            &namePattern,
 	}
 
-	if labels == nil {
-		labels = make(map[string]string)
-	}
-
+	labels := make(map[string]string)
```

Comment:
you can inline this map as follows:
```suggestion
	labels := map[string]string{
	  "bulk_batch": uuid.New().String()
	}
```